improver/utilities/mask.py

```diff
@@ -0,0 +1,66 @@
+# (C) Crown copyright, Met Office. All rights reserved.
+#
+# This file is part of IMPROVER and is released under a BSD 3-Clause license.
+# See LICENSE in the root of the repository for full licensing details.
+"""Module for applying mask to a cube."""
+
+import iris
+import numpy as np
+
+from improver.utilities.cube_checker import find_dimension_coordinate_mismatch
+from improver.utilities.cube_manipulation import (
+    enforce_coordinate_ordering,
+    get_coord_names,
+)
+
+
+def apply_mask(
+    cubes: iris.cube.CubeList, mask_name: str, invert_mask: bool = False
+) -> iris.cube.Cube:
+    """
+    Apply a provided mask to a cube. If invert_mask is True, the mask will be inverted.
+
+    Args:
+        cubes:
+            A list of iris cubes that should contain exactly two cubes: a mask and a cube
+            to apply the mask to. The cubes should have the same dimensions.
+        mask_name:
+            The name of the mask cube. It should match with exactly one of the cubes in
+            the input cubelist.
+        invert_mask:
+            If True, the mask will be inverted before it is applied.
+    Returns:
+        A cube with a mask applied to the data.
+
+    Raises:
+    ValueError: If the number of cubes provided is not equal to 2.
+
+    """
+    cubes = iris.cube.CubeList(cubes)
```

Comment:
Add import to the top of the module:
```
from improver.utilities.common_input_handle import as_cubelist
``` 
```suggestion
    cubes = as_cubelist(*cubes)
```

Reply:
Done